gef-ui/server/controllers/about-facility/index.test.js

```diff
@@ -100,7 +100,7 @@ describe('controllers/about-facility', () => {
     });
   });
 
-  describe('Validate About Facility', () => {
+  describe('POST About Facility', () => {
```

Comment:
I really dislike this pattern (which is present everywhere in the `server` folder), but i get why we've kept it. I also hate that update is a `put`.. I wouldn't say no to just updating this files `describes` to reference the functions being called instead.